frontend/src/v5/ui/components/shared/navigatonMenu/navigationMenu.component.tsx

```diff
@@ -42,11 +44,19 @@ export const NavigationMenu = ({ anchorEl, handleClose, list }: INavigationMenu)
 			horizontal: 'left',
 		},
 	};
+	const { project, teamspace } = useParams();
+	const selectedItem = project || teamspace;
```

Comment:
Can we move this to breadcrumbs.component and pass a selectedItem prop to this component?  
I think we might need to refactor that component (I dont believe is easy to read), but for now I think is better to have this decision there. Which is the component that 'knows' which option has been selected.

Reply:
Done